src/components/common/applicant-info-1.vue

```diff
@@ -42,16 +43,17 @@
                 :name="Math.random()"
                 autocomplete="chrome-off"
                 label="Middle Name (Optional)"
+                v-model="middleName"
```

Comment:
Updating the value was previously done by the `@input` event. Can you remove that event now (line 37) ?

Reply:
Actually, wait...

You have separate events for input changes and for blur. You also have, I think, 3 mechanisms for updating the properties (handleNameBlur, updateApplicant, and the v-model). This needs to be simplified.

Reply:
@input also calls clearValidation. Should I keep clearValidation? also, the @input be removed from firstName and lastName as well?

Reply:
At the moment, `@input` is what actually updates the values in the store, but you could do that differently.

Yes, it's the same problem with all 3 inputs.

Reply:
I'll respond in a different conversation. Closing this one.

Reply:
I've updated code by calling setApplicantDetails instead of using v-model. Please review again.